tests/integration/test_packages.py

```diff
@@ -34,14 +35,21 @@ def test_packages(env_package, env_name, test_env, tmpdir):
     """
     env_data = utils.load_test_data(f"{env_package}.yaml")[env_name]
     client = utils.Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
-    initial_response = client.create_new_request(
-        payload={
+
+    if env_name == "implicit_gomod":
+        payload = {
+            "repo": env_data["repo"],
+            "ref": env_data["ref"],
+        }
+    else:
+        payload = {
             "repo": env_data["repo"],
             "ref": env_data["ref"],
             "pkg_managers": env_data["pkg_managers"],
```

Comment:
How about just changing this line to 

`"pkg_managers": env_data.get("pkg_managers", []),`

and adding the following after the payload assignment:

```
if env_name == "implicit_gomod":
    payload.pop(pkg_managers)
```

Reply:
oh, I spend some time today trying to figure out how to simplify this but I didn't think of pop :D